packages/design-system/src/components/Clickable/Clickable.tsx

```diff
@@ -1,13 +1,13 @@
-import React, { forwardRef, ReactNode, ReactNodeArray, Ref, MouseEvent } from 'react';
+import React, { forwardRef, MouseEvent, ReactNode, Ref } from 'react';
 import classnames from 'classnames';
 import { Clickable as ReakitClickable, ClickableProps as ReakitClickableProps } from 'reakit';
 
 import styles from './Clickable.module.scss';
 
 export type ClickableProps = Omit<ReakitClickableProps, 'style'> & {
 	type?: 'button' | 'submit' | 'reset';
-	children: ReactNode | ReactNodeArray;
-	onClick: (event: MouseEvent<HTMLButtonElement> | KeyboardEvent) => void;
+	children: ReactNode | ReactNode[];
+	onClick?: (event: MouseEvent<HTMLButtonElement> | KeyboardEvent) => void;
```

Comment:
Not necessary, not in the HTML implem, and useless for some button types such as "submit" or "reset".